qubesbuilder/plugins/template/scripts/qubeize-image

```diff
@@ -66,13 +66,8 @@ trap cleanup ERR
 # ------------------------------------------------------------------------------
 mkdir -p "${ARTIFACTS_DIR}/qubeized_images/${TEMPLATE_NAME}"
 
-if [ "0$DISCARD_PREPARED_IMAGE" -eq "1" ]; then
-    echo "--> Moving $IMG to $IMG..."
-    mv "${BASE_IMG}" "$IMG" || exit 1
-else
-    echo "--> Copying $IMG to $IMG..."
-    cp "${BASE_IMG}" "$IMG" || exit 1
-fi
+echo "--> Copying $IMG to $IMG..."
+cp "${BASE_IMG}" "$IMG" || exit 1
```

Comment:
It should be `mv` here.

Reply:
In fact, I changed it again because it's the only possibility to copy-out prepared image before qubeization. Maybe we really don't care as currently we don't reuse it at all?